test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1

```diff
@@ -470,6 +472,8 @@ namespace PowershellTestConfigNamespace
                         $TestSessionConfigName = "TestRegisterPSSesionConfig"
                         Unregister-PSSessionConfiguration -Name $TestSessionConfigName -Force -NoServiceRestart -ErrorAction SilentlyContinue
                     }
+
+                    $expectedPSVersion = "$($PSVersionTable.PSVersion.Major)`.$($PSVersionTable.PSVersion.Minor)"
```

Comment:
I think this should be moved to the `BeforeAll` block of the `Describe`, otherwise the use of `$expectedPSVersion` below from a different `Context` block won't see the variable.

And the backtick is not necessary.

Reply:
Fixed.